docs/modules/processor/pages/logging-api.adoc

```diff
@@ -0,0 +1,90 @@
+= Logging
+
+All warning and error messages in Asciidoctor.js are routed through a logger.
+Many of the messages also include context about the source location (file, dir, path, lineno), which can be useful for integrations and tooling.
```

Comment:
My understanding is that if an error is not related to the input then the context about the source location will be absent.
For instance if an unexpected error occured at runtime.

Reply:
What about:

> Messages related to the input document also include context about the source location (...)

@mojavelinux Since I've took this sentence from the Asciidoctor 1.5.7 release note, what do you think about this ? Is it correct ? Is it clear enough ?